disperser/apiserver/disperse_blob_v2.go

```diff
@@ -32,6 +33,11 @@ func (s *DispersalServerV2) DisperseBlob(ctx context.Context, req *pb.DisperseBl
 		return nil, err
 	}
 
+	// Check against payment meter to make sure there is quota remaining
+	if err := s.checkPaymentMeter(ctx, req); err != nil {
+		return nil, err
```

Comment:
Shouldn't we be using api.NewErrorX for this return. Also noticed there are some other areas that aren't doing that.

Reply:
Isn't it already wrapped inside the function?

Reply:
Oh I see, seems like one level of indirection though. I was thinking the top level would handle that.

Reply:
yea the implementation here chose to let low level functions to classify the errors.

Reply:
actually like moving it up to have less redundant code. validateDispersalRequest should be homogeneous and all its errors are (and should be) invalid requests.